### PR TITLE
fix(themes): use only woff files and fix firefox font line gap warning

### LINKED ISSUE #2478: [Accordion] browser warning `downloadable font: OS/2: Bad sTypoLineGap...`
# Bug Report

## Steps to reproduce
1. open https://ui.agiletoolkit.org/demos/interactive/accordion.php (also reproducible on docs https://fomantic-ui.com/ homepage)
2. notice
   image
   in the browser console

tested with the latest Fomantic-UI beta and the problem is still present

browser: Firefox

reported also in https://github.com/Semantic-Org/Semantic-UI/issues/2146

I checked the browser console also for the Fomantic-UI docs and on https://fomantic-ui.com/modules/rating.html page (and many others) I found:

image

image

(probably doc unwanted log only)

in https://fomantic-ui.com/behaviors/visibility.html

image

(not exactly sure if these errors are expected)

in https://fomantic-ui.com/modules/sticky.html

image

link https://fomantic-ui.com/modules/dropdown.html#multiple-select-settings with fragment is not working - not navigating to the target position

### PR DESCRIPTION
**Pull request by lubber-de**
## Description
As  woff files are supported everwhere since IE11, i removed all eot,ttf,svg files and their references. Also adjusted some according theme references to match the new LESS-map icon generation

This PR also adjusts the negativ linegap issue. Thanks to @mvorisek which kindly added his fixes into this PR directly via https://github.com/lubber-de/Fomantic-UI/pull/51

## Closes
#2478 
#1254 
https://github.com/Semantic-Org/Semantic-UI/issues/2146

### CODE REVIEW

**Comment by mvorisek** [comment]
I am very sorry, the `truetype` removal does not fix the `downloadable font: OS/2: Bad sTypoLineGap...` console error, the error was only not shown after reload, full reload using ctrl+F5 is required to test properly. I am trying to figure out the real problem/solution.

**Comment by lubber-de** [comment]
> I am very sorry, the `truetype` removal does not fix the `downloadable font: OS/2: Bad sTypoLineGap...` console error, the error was only not shown after reload, full reload using ctrl+F5 is required to test properly. I am trying to figure out the real problem/solution.

Doesnt matter. THiS PR is still valid in case of not using ttf/eot/svg anymore. i'll adjust the PR description

File: src/themes/github/modules/dropdown.overrides [hunk 1]
```diff
@@ -12,7 +12,6 @@
...
...9xT3FAWPjo+NkIuQi5CJjogI9xT7FAWPh42Hi4aLhomHh4eIiIaJhosI+5SLBYaLh42HjoiPiY+LkAgO+92d928Vi5CNkI+OCPcU9xQFjo+QjZCLkIuPiY6Hj4iNhouGCIv7lAWLhomHh4iIh4eJhouGi4aNiI8I+xT3FAWHjomPi5AIDvvdi+YVi/eUBYuQjZCOjo+Pj42Qi5CLkImOhwj3FPsUBY+IjYaLhouGiYeHiAj7FPsUBYiHhomGi4aLh42Hj4iOiY+LkAgO+JQU+JQViwwKAAAAAAMCAAGQAAUAAAFMAWYAAABHAUwBZgAAAPUAGQCEAAAAAAAAAAAAAAAAAAAAARAAAAAAAAAAAAAAAAAAAAAAQAAA8NoB4P/g/+AB4AAgAAAAAQAAAAAAAAAAAAAAIAAAAAAAAgAAAAMAAAAUAAMAAQAAABQABAA4AAAACgAIAAIAAgABACDw2v/9//8AAAAAACDw1//9//8AAf/jDy0AAwABAAAAAAAAAAAAAAABAAH//wAPAAEAAAABAAA5emozXw889QALAgAAAAAA0ABHWAAAAADQAEdYAAAAAAElAW4AAAAIAAIAAAAAAAAAAQAAAeD/4AAAAgAAAAAAASUAAQAAAAAAAAAAAAAAAAAAAAgAAAAAAAAAAAAAAAABAAAAASUAAAElAAAAtwASALcAAAAAUAAACAAAAAAADgCuAAEAAAAAAAEADgAAAAEAAAAAAAIADgBHAAEAAAAAAAMADgAkAAEAAAAAAAQADgBVAAEAAAAAAAUAFgAOAAEAAAAAAAYABwAyAAEAAAAAAAoANABjAAMAAQQJAAEADgAAAAMAAQQJAAIADgBHAAMAAQQJAAMADgAkAAMAAQQJAAQADgBVAAMAAQQJAAUAFgAOAAMAAQQJAAYADgA5AAMAAQQJAAoANABjAGkAYwBvAG0AbwBvAG4AVgBlAHIAcwBpAG8AbgAgADEALgAwAGkAYwBvAG0AbwBvAG5pY29tb29uAGkAYwBvAG0AbwBvAG4AUgBlAGcAdQBsAGEAcgBpAGMAbwBtAG8AbwBuAEYAbwBuAHQAIABnAGUAbgBlAHIAYQB0AGUAZAAgAGIAeQAgAEkAYwBvAE0AbwBvAG4ALgAAAAADAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA) format('woff')
```
Comment (mvorisek) [inline src/themes/github/modules/dropdown.overrides:15]: this data URI/font is 1:1 duplicated with font in `default` theme - is it possible to remove it here?

only `Dropdown` font:

image

other data URI fonts are not duplicated/fine

Comment (lubber-de) [reply src/themes/github/modules/dropdown.overrides:15]: I think it is necessary as the override files are theme specific. So the default dropdown.overrides will not be loaded when the github theme was selected in theme.config....

**Review by exoego — APPROVED** [review]
https://caniuse.com/woff

LGTM

**Pull request merged**